### ISSUE TITLE
Chronos 3.0.0 for Ubuntu 16.04

### ISSUE THREAD

**Issue by dddpaul** [mesos/chronos#792]
Hello.

Will Chronos 3.0.0 be published in Mesosphere repository for current Ubuntu LTS (16.04)?

**Comment by brndnmtthws** [comment]
No, I'm not going to maintain Debian packages. I've only got the time/energy to publish Docker images. Unless someone else wants to do it, it probably won't happen.

I'll leave this issue open to collect feedback.

**Comment by kpashka** [comment]
@brndnmtthws Why aren't you guys publishing jar-files in GitHub `Release` section? Feels like you're already passing them from CI to Docker during container build - https://github.com/mesos/chronos/blob/master/Dockerfile#L14
